### PR TITLE
[v6r20] fix monitoring integration test

### PR DESCRIPTION
**Pull request by zmathe**
BEGINRELEASENOTES
*tests
FIX: add MonitoringDB to the configuration
ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
I think "localhost" and "9200" are the default... is it necessary at all?

**Comment by zmathe** [comment]
The Monitoring test is failing because the db section is missing, because of this.

**Comment by fstagni** [comment]
Then I believe that you can revert this specific change: https://github.com/DIRACGrid/DIRAC/pull/3609/files#diff-f1a14cc3d28f4edc9b5f3fd5b040fdd7 and then there would be no need to change tests/Jenkins/dirac-cfg-update-dbs.py as you did here.

**Comment by fstagni** [comment]
Anyway, check the Travis build.

**Comment by zmathe** [comment]
I added the NoSQL part to the cfg and also fixed travis. I will commit as soon as autopep8 works perfectly :)

File: tests/Jenkins/dirac-cfg-update-dbs.py [hunk 1]
```diff
@@ -5,10 +5,14 @@
 import os
 
 from DIRAC.Core.Base import Script
+from DIRAC.FrameworkSystem.private.monitoring.MonitoringCatalog import MonitoringCatalog
+from sqlalchemy import databases
+from DIRAC.MonitoringSystem.Client.ServerUtils import monitoringDB
+from idlelib.rpc import LOCALHOST
```
Comment (fstagni) [inline tests/Jenkins/dirac-cfg-update-dbs.py]: Do you need these imports here?

File: AccountingSystem/Client/Types/Job.py [hunk 2]
```diff
@@ -1,60 +1,60 @@
 # $HeadURL$
```
Comment (fstagni) [inline AccountingSystem/Client/Types/Job.py]: Please remove all these `# $HeadURL$` wherever you see them.

File: tests/Jenkins/dirac-cfg-update-dbs.py (hunk 1 above)
Comment (zmathe) [reply tests/Jenkins/dirac-cfg-update-dbs.py]: no need

File: AccountingSystem/Client/Types/Job.py (hunk 2 above)
Comment (zmathe) [reply AccountingSystem/Client/Types/Job.py]: done

**Pull request merged**